### PR TITLE
Added information about the connection string parameters to README.md

### PR DESCRIPTION
**Pull request by johnfercher**
Added CLion junk files folder to gitignore, added section in README.md to inform the possible connection string parameters.

I hope that new users doesn't have doubt about which parameters use in future, like I were in #76.

### CODE REVIEW

**Comment by steffengy** [comment]
Those aren't parameter values but parameter aliases.
I think it'd be better to do something like maybe linking to [microsoft documentation](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring?redirectedfrom=MSDN&view=netframework-4.7.2#System_Data_SqlClient_SqlConnection_ConnectionString)

and adding a list of supported parameter aliases to the README.

File: README.md [hunk 1]
```diff
@@ -58,7 +58,22 @@ type please create an issue, which will make this table grow.
...
-
 Support for versions below 7.4 (to 7.2 so that everything >= SQL Server 2008 works) is desired.
+
+### Supported Connection String Parameters
+
+Not every parameter is supported yet. Below is a list of all parameters supported.
+
+|Parameter|Aliases|Description|
+|--------|--------|--------|
+|server|server|The name or network address of the instance of SQL Server to which to connect. The port number can be specified after the server name:|
```
[the remark this answers is not in the document]
Comment (johnfercher) [reply README.md]: I pushed this modification to the branch.

File: README.md [hunk 2]
```diff
@@ -58,7 +58,22 @@ type please create an issue, which will make this table grow.
...
+|integratedsecurity|integratedsecurity|When `false`, User ID and Password are specified in the connection. When `true`, the current Windows account credentials are used for authentication.|
+|username|uid, username, user|The SQL Server login account. Not recommended. To maintain a high level of security, we strongly recommend that you use the `Integrated Security` or `Trusted_Connection` keywords instead. [SqlCredential](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlcredential?view=netframework-4.7.2) is a more secure way to specify credentials for a connection that uses SQL Server Authentication.|
```
[the remark this answers is not in the document]
Comment (johnfercher) [reply README.md]: I think it whould be nice if we have a short description about the parameter.

**Comment by steffengy** [comment]
Thanks!

**Pull request merged**